website/docs/r/cis_filter.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+
+subcategory: "Internet services"
+layout: "ibm"
+page_title: "IBM: ibm_cis_filter"
+description: |-
+  Provides a IBM CIS Filter.
+---
+
+# ibm_cis_filter
+
+Provides a IBM CIS Filter. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to create, update, delete filter of a domain of a CIS instance
+
+## Example Usage
+
+```terraform
+# Add a filter to the domain
+
+resource "ibm_cis_filter" "test" {
+  cis_id      = data.ibm_cis.cis.id
+  domain_id   = data.ibm_cis_domain.cis_domain.domain_id
+  expression  =  "(http.request.uri eq \"/test-update?number=212\")"
+  paused      =  "false"
+  description = "Filter-creation"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `cis_id` - (Required,string) The ID of the CIS service instance
+- `domain_id` - (Required,string) The ID of the domain to add the Filter.
+- `expression` - (Required,string) The expression of filter.
+- `paused` - (Optional,boolean). Whether this filter is currently disabled.
+- `description` - (Required,string) Some useful information about this filter to help identify the purpose of it.
```

Comment:
According to resouce `description is optional.. Please make arguments inline to datasource`